REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 1.0
 FEMBase
-Tensors
```

Comment:
The plan is to use Tensors.jl in this package.

Reply:
Dropped Tensors due to interface error. Might also be just code bug ;D. Anyways, abandoning PR, due Hackaton changes.